OutOfSchool/OutOfSchool.DataAccess/OutOfSchoolDbContext.SoftDelete.cs

```diff
@@ -58,7 +58,7 @@ private void ApplySoftDelete(ModelBuilder builder)
             .ApplySoftDelete<CompanyInformation>()
             .ApplySoftDelete<CompanyInformationItem>()
             .ApplySoftDelete<DateTimeRange>()
-            .ApplySoftDelete<Direction>()
+            //.ApplySoftDelete<Direction>()
```

Comment:
I'm not insisting, but maybe it's right time to remove this commented out code?

Reply:
We planned to clear it after finishing changing all entities. Commented lines show ours progress )))

Reply:
Code as a backlog? I'll steal this idea but would never ever use it )
And thanks for explanation